src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
+  const deprecatedMessage = deprecatedCard && plugin.deprecatedMessage;
+  const cardClasses = classnames('card', { [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false });
+  const cardHeader = classnames( 'card__header', deprecatedCard ? styles.deprecatedCardHeader : styles.pluginCardHeader );
+  const cardBody = classnames( deprecatedCard ? styles.pluginCardFlagMessage : 'card__body' );
```

Comment:
You don't need to use `classnames()` if you only have 1 className string

Reply:
I review the classnames concept through the npm official page and the blogs.